pkg/api/apis/operators/v1alpha1/clusterserviceversion_types.go

```diff
@@ -65,9 +66,10 @@ type CRDDescription struct {
 
 // APIServiceDescription provides details to OLM about apis provided via aggregation
 type APIServiceDescription struct {
-	Name              string                 `json:"name"`
+	Group             string                 `json:"group"`
 	Version           string                 `json:"version"`
 	Kind              string                 `json:"kind"`
+	DeploymentName    string                 `json:"deploymentName,omitempty"`
```

Comment:
I wonder if this should accept a container as well? `deploymentName: packages-apiserver/pkg-container`